.github/workflows/ci.yml

```diff
@@ -44,52 +47,50 @@ jobs:
           sed -i "17i sed -i 's|pkgs|pkgs-test|g' /etc/apt/apt.conf.d/90pkgs-nginx" docker/Dockerfile
           sed -i 's|deb https|deb [trusted=yes] https|g' docker/Dockerfile
           sed -i 's|\${NGINX_PLUS_VERSION^^}/||g' docker/Dockerfile
+
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+        uses: docker/setup-buildx-action@4c0219f9ac95b02789c1075625400b2acbff50b1 # v2.9.1
+
       - name: Build Plus Docker Image
-        uses: docker/build-push-action@v4
+        uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825 # v4.1.1
         with:
           file: docker/Dockerfile
-          context: 'docker'
+          context: "docker"
           tags: nginx-plus:${{ env.NGINX_PLUS_VERSION }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
           load: true
           secrets: |
             "nginx-repo.crt=${{ secrets.NGINX_CRT }}"
             "nginx-repo.key=${{ secrets.NGINX_KEY }}"
-          build-args:
-            NGINX_PLUS_VERSION=${{ env.NGINX_PLUS_VERSION }}
+          build-args: NGINX_PLUS_VERSION=${{ env.NGINX_PLUS_VERSION }}
+
       - name: Test Client
         if: (github.event_name != 'schedule')
         run: make test
+
       - name: Test Client (Nightly)
         if: (github.event_name == 'schedule')
         run: make test
         env:
           NGINX_PLUS_VERSION: nightly
 
-  release:
-    name: Release
-    runs-on: ubuntu-22.04
-    if: ${{ startsWith(github.ref, 'refs/tags/') }}
-    needs: build
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v3
+      - name: Create/Update Draft
+        uses: lucacome/draft-release@f6dc37dcdf44be100a649b72c62c628776750190 # v0.2.2
```

Comment:
I noticed this step is using a `draft-release` image from your personal repo. Is this okay?

Reply:
Yes, I created a GitHub action a few months ago to simplify our release process 🙂